packages/types/src/user.ts

```diff
@@ -6,4 +6,13 @@ export const UserSchema = type({
     userData: ScheduleSaveStateSchema,
 });
 
+export const AuthUserSchema = type({
+    id: 'string',
+    'userData?': ScheduleSaveStateSchema,
+    name: 'string',
```

Comment:
Is this `name` their name on google or the name of their userData object? Can we document that somewhere?

Reply:
When using the combined schema, the user's selected name should be under "id", so this field shouldn't be necessary.